www/src/css/searchbox-style.css

```diff
@@ -0,0 +1,935 @@
+.ais-InstantSearch__root {
```

Comment:
This all should be done with Glamor using the `css` prop.

Reply:
Actually... not even sure what this is

Reply:
This file looks like it was scss or less transpiled to css. Is it worth refactoring over into Glamor or leave it be?

Reply:
First question is double-check it's even being used. It's not referenced anywhere in this PR

Reply:
I'm not sure why it's not referenced but as far as I know this file is only on this branch. It's adding a lot of styles used here: https://yarnpkg.com/en/ on things like the search bar. Some of the styles aren't really important so I'm sorting through it and I'll see if some could be glamor-ified.

Reply:
They're not imported anywhere so they're not used so just delete the file.

Reply:
It looks like it was imported to the /layouts/index.js file. I took out references to classes that weren't styling anything so it's a little shorter, but deleting the file left things like the search bar in their natural-plain-browser state.

Reply:
I cut out like ~300ish lines of styles here, there are like 500 lines that are applied to components that are used. I think the easiest way to add them in as component props would be convert the css back to scss and then use those styles adding them as props to the react instant search components we're using. Should I go through and do that? Is there a need to style them that way over the stylesheet?

Reply:
no it's ok for now I guess. Feels like an absurd amount of CSS but whatever. We can refactor it later.